nautobot/dcim/api/serializers.py

```diff
@@ -389,6 +389,7 @@ class RackElevationDetailFilterSerializer(serializers.Serializer):
     expand_devices = serializers.BooleanField(required=False, default=True)
     include_images = serializers.BooleanField(required=False, default=True)
     display_fullname = serializers.BooleanField(required=False, default=True)
+    is_occupied = serializers.BooleanField(required=False, allow_null=True, default=None)
```

Comment:
Still needed?

Reply:
I no longer need it for the device form, but it's still a valid API filter that _could_ be used in the future. I can remove it if you want though to keep the scope of this PR simple.

Reply:
Fair enough. :-)